_pages/en_US/f3xswift-(mac).txt

```diff
@@ -1,33 +1,32 @@
 ---
-title: "F3X (Mac)"
+title: "F3XSwift (Mac)"
 ---
 
 {% include toc title="Table of Contents" %}
 
 ### Required Reading
 
-This is an add-on section for checking your SD card for errors using F3X.
+This is an add-on section for checking your SD card for errors using F3XSwift.
 
 Depending on the size of your SD card and the speed of your computer, this process can take up to several hours!
 
 This page is for Mac users only. If you are not on Mac, check out the [H2testw (windows)](h2testw-(windows)) or [F3 (Linux)](f3-(linux)) pages.
 
 ### What You Need
 
-* The latest release of [F3X](https://github.com/insidegui/F3X/releases/latest)
+* The latest release of [F3XSwift](https://github.com/vrunkel/F3XSwift/releases/latest)
 
 ### Instructions
 
-1. Unzip the F3X `.zip` file
+1. Unzip the F3XSwift `.zip` file
 1. Insert your SD card into your computer
-1. Run the F3X app
+1. Run the F3XSwift app
 1. Select your SD card
-1. Press "Start Test"
-1. Wait until the process is complete.
+1. Select “Test”
+1. Select "Open"
```

Comment:
These instructions are a bit vague. It should be specified what to test (the SD card -- though, macOS should automatically gray out the user data and any windows partitions) and when to click open "when you see the finder window open" or something).